plugins/ci/pkg/models/models.go

```diff
@@ -116,7 +116,7 @@ func maybeAddSlash(input string) string {
 // SetDefaults sets configurationd defaults
 func (c *Configuration) SetDefaults() {
 	if c.Options.TempFolder == "" {
-		c.Options.TempFolder = "./_insightsTemp/"
+		c.Options.TempFolder = "/tmp/"
```

Comment:
I feel like it might be safer to have `/tmp/<something>` like maybe still keep the _insightsTemp name, but just inside `/tmp/` instead of `.`